spec.py

```diff
@@ -280,6 +295,11 @@ def reaction(self, r):
             self.product_1 = 'p' # Since the proton is a charged particle, as of now, only an energy spectrum for 4*pi emission can be returned
             self.product_2 = '4he'
 
+        elif all(p in [a,b] for p in ['4he','9be']): #!#
+            # 9Be(4He,12C)n reaction
+            self.product_1 = '12c_'+self.product_state
```

Comment:
Maybe this will result in an error. Product state can be "GS", "1L" or "2L" right? But the species names are "12c_gs", "12c_1l" and "12c_2l". That is, the "GS" and "L" are uppercase in the function input variable names, but is lowercase in the particle name definitions. I don't know if Python discriminates uppercase and lowercase by default? It should not? In that case, you should change line 300 to 
`self.product_1 = '12c_'+self.product_state.upper()`. Or, maybe better, on line 255, you should always do 
`self.product_state = product_state.lower()`